p2-layouts-master/code/style.css

```diff
@@ -0,0 +1,209 @@
+/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+
+body {
+  background-color: rgb(255, 248, 213);
+}
+
+header {
+  background-image: url('watermelon.JPG');
+  background-size: cover;
+  padding: 100px 0 100px 0;
+  text-align: center;
+}
+
+.header-text {
+  padding: 50px;
+  margin-top: 10px;
+  max-width: 500px;
+  margin: 0 auto;
+  color: white;
+}
+
+h1 {
+  display: right;
+  padding: 10px;  
+  margin-top: -100px;
+  margin-left: -200px;
+  float:right;   
+}
+
+h2 {
+  display: left;
+  padding: 10px;  
+  margin-top: -40px;
+  margin-left: -200px;
+  float:left;
+}
+
+nav {    
+  padding: 50px;
+  margin-top: 10px;    
+  margin: auto;
+  width: 100%;
+  display: flex;
+  justify-content: center;
```

Comment:
This makes the navbar very squished together. Consider changing the value to `space-between` or adding some margin to the sides of the `a`'s